packages/store/src/decorators/state.ts

```diff
@@ -24,12 +29,12 @@ export function State<T>(options: StoreOptions<T>) {
     meta.name = name;
   }
 
-  return function(target: StateClass) {
+  return (target: StateClass): void => {
```

Comment:
Because we don't use void on a regular basis, I think we should avoid adding that.

Reply:
I think it's time for us to use return types everywhere in the future to configure tslint